File: Fore-C_v2/codes/Final_covariates_by_disease_and_region.py
# -*- coding: utf-8 -*-
"""
Created on Thu Dec  9 12:16:50 2021

@author: jamie
"""

import joblib

# open final model objects
GA_GBR_Model = joblib.load("../model_objects/ga_gbr.joblib")
GA_Pacific_Model = joblib.load("../model_objects/ga_pac.joblib")
WS_GBR_Model = joblib.load("../model_objects/ws_gbr.joblib")
WS_Pacific_Model = joblib.load("../model_objects/ws_pac_acr.joblib")

# Growth anomalies GBR 
ga_gbr_vars = GA_GBR_Model.feature_names_in_

# Growth anomalies Pacific
ga_pac_vars = GA_Pacific_Model.feature_names_in_

# White syndreoms GBR 
ws_gbr_vars = WS_GBR_Model.feature_names_in_

# White syndromes Pacific (Acroporidae)
ws_pac_acr_vars = WS_Pacific_Model.feature_names_in_

File: Fore-C_v2/codes/Final_covariates_by_disease_and_region.R
# Final co-variates in models for each disease and region

# open final model objects
GA_GBR_Model <- readRDS("../model_objects/ga_gbr.rds")
GA_Pacific_Model <- readRDS("../model_objects/ga_pac.rds")
WS_GBR_Model <- readRDS("../model_objects/ws_gbr.rds")
WS_Pacific_Model <- readRDS("../model_objects/ws_pac.rds")

# Growth anomalies GBR 
ga_gbr_vars <- names(GA_GBR_Model$importance[,1]) 

# Growth anomalies Pacific
ga_pac_vars <- names(GA_Pacific_Model$importance[,1])

# White syndromes Pacific (Acroporidae)
ws_pac_acr_vars <- names(WS_Pacific_Model$importance[,1]) 

# White syndromes GBR 
ws_gbr_vars <- names(WS_GBR_Model$importance[,1]) 
